tests/dragonfly/json_test.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+from redis.commands.json.path import Path
+from .utility import *
+
+jane = {
+    'name': "Jane",
+    'Age': 33,
+    'Location': "Chawton"
+}
+
+json_num = {
+    "a":{"a":1, "b":2, "c":3}
+}
+
+def get_type(connection, key):
+    return connection.execute_command("type", key)
+
+def get_set_json(connection, key, value, path="$"):
+
+    try:
+        connection.json().set(key, path, value)
+
+        result = connection.json().get(key, path)
+        return result
+    except redis.exceptions.ResponseError as re:
+        return str(re)
+    except Exception as e:
+        return None
+
+def get_str(connect, key):
+    try:
+        return connect.get(key)
+    except redis.exceptions.ResponseError as re:
+        return str(re)
+    except Exception as e:
+        return None
+
+def test_basic_json_get_set(client):
+    key_name="test-json-key"
+    result = get_set_json(connection=client, key=key_name, value=jane)
+    if result is None:
+        assert False, "failed to set JSON value"
+    else:
```

Comment:
You can always just do `assert result is not None, "...."` and continue linear flow

Reply:
You're right - thanks (this what's happen when coping code from some offline test :) )